geoportal/geoportailv3_geoportal/static-ngeo/js/apps/MainController.js

```diff
@@ -1969,6 +1971,19 @@ MainController.prototype.toggleThemeSelector = function() {
     themesSwitcher.collapse('show');
     layerTree.collapse('hide');
   }
+
+  if (!layersOpen.value) {
+    useAppStore().setLayersOpen(true)
+    myLayersTabOpen.value && useAppStore().setMyLayersTabOpen(false)
+    useAppStore().setThemeGridOpen(true)
+  } else if (layersOpen.value) {
+    if (themeGridOpen.value) {
+      useAppStore().setLayersOpen(false)
```

Comment:
I think when another tool in the footer bar is open and the themegrid has been opened before (and still has an open state), the code can fall into this condition and close the sidepanel instead of showing the themegrid.

Reply:
yes, thanks for reporting this, I have made a fix.

Reply:
Thanks for the fix, works as expected now :-)